# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../transfig/usr-local.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -ur transfig.3.2.5/fig2dev/dev/Makefile transfig.3.2.5-fixed/fig2dev/dev/Makefile
--- transfig.3.2.5/fig2dev/dev/Makefile	2005-08-18 16:31:10.000000000 +0000
+++ transfig.3.2.5-fixed/fig2dev/dev/Makefile	2007-05-09 12:53:27.000000000 +0000
@@ -868,40 +868,40 @@
 install::
 
 install:: japanese.ps
-	@if [ -d $(DESTDIR)/usr/local/lib/fig2dev ]; then \
+	@if [ -d $(DESTDIR)$(XFIGLIBDIR)/fig2dev ]; then \
 	set +x; \
 	else \
-	if [ -h $(DESTDIR)/usr/local/lib/fig2dev ]; then \
-	(set -x; rm -f $(DESTDIR)/usr/local/lib/fig2dev); \
+	if [ -h $(DESTDIR)$(XFIGLIBDIR)/fig2dev ]; then \
+	(set -x; rm -f $(DESTDIR)$(XFIGLIBDIR)/fig2dev); \
 	fi; \
-	(set -x; $(MKDIRHIER) $(DESTDIR)/usr/local/lib/fig2dev); \
+	(set -x; $(MKDIRHIER) $(DESTDIR)$(XFIGLIBDIR)/fig2dev); \
 	fi
-	$(INSTALL) $(INSTALLFLAGS) $(INSTDATFLAGS) japanese.ps $(DESTDIR)/usr/local/lib/fig2dev/japanese.ps
-	( cd $(DESTDIR)/usr/local/lib/fig2dev; 	  rm -f ja.ps ja_JP.ps ja_JP.ujis.ps ja_JP.eucJP.ps ja_JP.EUC.ps; 	  $(LN) japanese.ps ja.ps; $(LN) japanese.ps ja_JP.ps; 	  $(LN) japanese.ps ja_JP.ujis.ps; $(LN) japanese.ps ja_JP.eucJP.ps; 	  $(LN) japanese.ps ja_JP.EUC.ps )
+	$(INSTALL) $(INSTALLFLAGS) $(INSTDATFLAGS) japanese.ps $(DESTDIR)$(XFIGLIBDIR)/japanese.ps
+	( cd $(DESTDIR)$(XFIGLIBDIR)/fig2dev; 	  rm -f ja.ps ja_JP.ps ja_JP.ujis.ps ja_JP.eucJP.ps ja_JP.EUC.ps; 	  $(LN) japanese.ps ja.ps; $(LN) japanese.ps ja_JP.ps; 	  $(LN) japanese.ps ja_JP.ujis.ps; $(LN) japanese.ps ja_JP.eucJP.ps; 	  $(LN) japanese.ps ja_JP.EUC.ps )
 
 install:: korean.ps
-	@if [ -d $(DESTDIR)/usr/local/lib/fig2dev ]; then \
+	@if [ -d $(DESTDIR)$(XFIGLIBDIR)/fig2dev ]; then \
 	set +x; \
 	else \
-	if [ -h $(DESTDIR)/usr/local/lib/fig2dev ]; then \
-	(set -x; rm -f $(DESTDIR)/usr/local/lib/fig2dev); \
+	if [ -h $(DESTDIR)$(XFIGLIBDIR)/fig2dev ]; then \
+	(set -x; rm -f $(DESTDIR)$(XFIGLIBDIR)/fig2dev); \
 	fi; \
-	(set -x; $(MKDIRHIER) $(DESTDIR)/usr/local/lib/fig2dev); \
+	(set -x; $(MKDIRHIER) $(DESTDIR)$(XFIGLIBDIR)/fig2dev); \
 	fi
-	$(INSTALL) $(INSTALLFLAGS) $(INSTDATFLAGS) korean.ps $(DESTDIR)/usr/local/lib/fig2dev/korean.ps
-	( cd $(DESTDIR)/usr/local/lib/fig2dev; 	  rm -f ko.ps ko_KR.ps ko_KR.eucKR.ps ko_KR.EUC.ps; 	  $(LN) korean.ps ko.ps; $(LN) korean.ps ko_KR.ps; 	  $(LN) korean.ps ko_KR.eucKR.ps; $(LN) korean.ps ko_KR.EUC.ps )
+	$(INSTALL) $(INSTALLFLAGS) $(INSTDATFLAGS) korean.ps $(DESTDIR)$(XFIGLIBDIR)/fig2dev/korean.ps
+	( cd $(DESTDIR)$(XFIGLIBDIR)/fig2dev; 	  rm -f ko.ps ko_KR.ps ko_KR.eucKR.ps ko_KR.EUC.ps; 	  $(LN) korean.ps ko.ps; $(LN) korean.ps ko_KR.ps; 	  $(LN) korean.ps ko_KR.eucKR.ps; $(LN) korean.ps ko_KR.EUC.ps )
 
 install:: cs_CZ.ps
-	@if [ -d $(DESTDIR)/usr/local/lib/fig2dev ]; then \
+	@if [ -d $(DESTDIR)$(XFIGLIBDIR)/fig2dev ]; then \
 	set +x; \
 	else \
-	if [ -h $(DESTDIR)/usr/local/lib/fig2dev ]; then \
-	(set -x; rm -f $(DESTDIR)/usr/local/lib/fig2dev); \
+	if [ -h $(DESTDIR)$(XFIGLIBDIR)/fig2dev ]; then \
+	(set -x; rm -f $(DESTDIR)$(XFIGLIBDIR)/fig2dev); \
 	fi; \
-	(set -x; $(MKDIRHIER) $(DESTDIR)/usr/local/lib/fig2dev); \
+	(set -x; $(MKDIRHIER) $(DESTDIR)$(XFIGLIBDIR)/fig2dev); \
 	fi
-	$(INSTALL) $(INSTALLFLAGS) $(INSTDATFLAGS) cs_CZ.ps $(DESTDIR)/usr/local/lib/fig2dev/cs_CZ.ps
-	( cd $(DESTDIR)/usr/local/lib/fig2dev; 	  rm -f hr_HR.ps hu_HU.ps pl_PL.ps ro_RO.ps sk_SK.ps sl_SI.ps; 	  $(LN) cs_CZ.ps hr_HR.ps; $(LN) cs_CZ.ps hu_HU.ps; 	  $(LN) cs_CZ.ps ro_RO.ps; $(LN) cs_CZ.ps pl_PL.ps; 	  $(LN) cs_CZ.ps sk_SK.ps; $(LN) cs_CZ.ps sl_SI.ps )
+	$(INSTALL) $(INSTALLFLAGS) $(INSTDATFLAGS) cs_CZ.ps $(DESTDIR)$(XFIGLIBDIR)/fig2dev/cs_CZ.ps
+	( cd $(DESTDIR)$(XFIGLIBDIR)/fig2dev; 	  rm -f hr_HR.ps hu_HU.ps pl_PL.ps ro_RO.ps sk_SK.ps sl_SI.ps; 	  $(LN) cs_CZ.ps hr_HR.ps; $(LN) cs_CZ.ps hu_HU.ps; 	  $(LN) cs_CZ.ps ro_RO.ps; $(LN) cs_CZ.ps pl_PL.ps; 	  $(LN) cs_CZ.ps sk_SK.ps; $(LN) cs_CZ.ps sl_SI.ps )
 
 # ----------------------------------------------------------------------
 # common rules for all Makefiles - do not edit
